src/components/ReposList.module.scss

```diff
@@ -1,12 +1,13 @@
 .container {
     color: white;
-    flex:1 1 0;
-    width: 0;
+    // flex:1 1 0;
+    // width: 0;
```

Comment:
```suggestion
```

Reply:
if you don't want to remove these lines you can add a comment above explaining why